public/app/plugins/datasource/influxdb/components/VisualInfluxQLEditor/FromSection.tsx

```diff
@@ -35,7 +35,7 @@ export const FromSection = ({
     // if `default` does not exist in the list of policies, we add it
     const allPoliciesWithDefault = allPolicies.some((p) => p === 'default')
       ? allPolicies
-      : [DEFAULT_POLICY, allPolicies];
+      : [DEFAULT_POLICY, ...allPolicies];
 
     return allPoliciesWithDefault.map(toSelectableValue);
```

Comment:
at this point typescript should have catched it, `toSelectableValue` needs a `string`. i will investigate later why it did not catch it.

Reply:
nevermind, found the reason. we are not running currently with `strict:true` in the `tsconfig.json` file, so not all checks happen.